components/apimgt/org.wso2.carbon.apimgt.internal.service/swagger.json

```diff
@@ -745,6 +752,31 @@
         }
       }
     },
+    "/apis/undeployed-revision" : {
```

Comment:
why was this endpoint added?

Reply:
Fixed via https://github.com/wso2/carbon-apimgt/pull/11241/commits/2df6a7918cedae735bf9d306297b7fb82bbbcf71